app/services/import_planned_disbursements.rb

```diff
@@ -0,0 +1,37 @@
+require "csv"
+
+class ImportPlannedDisbursements
+  RODA_ID_KEY = "RODA identifier"
+  FORECAST_COLUMN_HEADER = /FC +(\d{4})\/\d{2} +FY +Q([1-4])/
+
+  def initialize(report:)
+    @report = report
+  end
+
+  def import(forecasts)
+    forecasts.each { |row| import_row(row) }
+  end
+
+  def import_row(row)
+    roda_identifier = row[RODA_ID_KEY]
+    activity = Activity.by_roda_identifier(roda_identifier)
+
+    row.each do |key, value|
+      match = FORECAST_COLUMN_HEADER.match(key)
+      next unless match
```

Comment:
Should we raise an error if a column doesn't match this pattern? (assuming we throw away and use the RODA ID)

Reply:
We can't do that in this case because the input files contain a lot of columns we should ignore -- they're there for informational purposes to make compiling the data easier. I did wonder about how to avoid situations where a column is intended to contain forecast values but they end up being ignored b/c they don't match this pattern.

Reply:
👍